crates/rattler_installs_packages/src/artifacts/sdist.rs

```diff
@@ -43,6 +44,25 @@ pub enum SDistError {
     WheelCoreMetaDataError(#[from] WheelCoreMetaDataError),
 }
 
+/// Pop until the file name is found an use the rest of the path
```

Comment:
If I understand the function correctly, then the description is not quite matching what it does, right? `name` -> `prefix` and maybe something like:

`Returns the remainder of the path that follows the last component that starts with "prefix". For example, `strip_until_containing_prefix("bla", "/my/blabli/path/bla/path/remainder) -> "path/remainder"`

Reply:
Yes that seems better!